src/ui/atoms/icon.js

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import PropTypes from "prop-types"
+// When you need more icons just add svg-path in getIconPath and call it
+// with <Icon name="yourname" />
+
+const getIconPath = (name, props) => {
```

Comment:
I think we can use [`svg-react-loader`](https://github.com/jhamlet/svg-react-loader)
to load real .svg icons to react

Reply:
I am not satisfied with a way Icon component works now, especially hovers. I am going to fix/change work with svg in another branch/pr.